bandit/core/docs_utils.py

```diff
@@ -3,7 +3,10 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 # where our docs are hosted
-BASE_URL = "https://bandit.readthedocs.io/en/latest/"
+import bandit
+
+
+BASE_URL = "https://bandit.readthedocs.io/"
```

Comment:
Why not do something like:

```py
BASE_URL = f"https://bandit.readthedocs.io/en/{bandit.__version__}/"
```

And then keep the rest of the code simpler?

Reply:
Makes sense, will change it.

Reply:
Hmm, not sure why, but using bandit.__version__ as a global here causes stevedore to break to load any extensions.

Reply:
I added a failure callback to the extension loader of Stevedore and get this
```module 'bandit' has no attribute '__version__'```
So root cause is that because BASE_URL is a global and calling into bandit module before it has been loaded, 
there is no such attribute.

Reply:
Think I can fix a few ways:
1. revert to previous change
2. move BASE_URL into the get_url() function. Seems BASE_URL is only global for the unit test purposes
3. use pbr.version.VersionInfo("bandit").version_string() instead of bandit.__version__

Reply:
I went with option 2.